valohai/internals/yaml.py

```diff
@@ -70,19 +69,30 @@ def parse_input(key: str, value: Any) -> Input:
         value["name"] = key
         return Input.parse(value)
 
+    # Check and ignore the local path file
+    if isinstance(value, List):
```

Comment:
I think this should be
```suggestion
    if isinstance(value, list):
```
(using `typing.List`, as here, will apparently work due to some _magic_, but it's not idiomatic.)